modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -2002,10 +1999,12 @@ private static CompletableFuture<List<BinaryRow>> collectMultiRowsResponsesWithR
     protected CompletableFuture<IgniteBiTuple<ClusterNode, Long>> enlist(int partId, InternalTransaction tx) {
         HybridTimestamp now = tx.startTimestamp();
 
-        TablePartitionId tablePartitionId = new TablePartitionId(tableId, partId);
-        tx.assignCommitPartition(tablePartitionId);
+        ReplicationGroupId replicationGroupId = targetReplicationGroupId(partId);
+
+        // TODO sanpwc ?
```

Comment:
A TODO without an issue reference

Reply:
Removed. I've considered an option to change commitPartitionId type to ZonePartitionId - but gave up this idea.